src/encoded/static/components/item-pages/FileView.js

```diff
@@ -30,32 +30,30 @@ export default class FileView extends WorkflowRunTracingView {
         );
     }
 
-    static shouldHiGlassViewExist(context){
-        // TODO: Remove context.file_format check?
-        if (!context.higlass_uid || typeof context.higlass_uid !== 'string') return false;
-        var fileFormat  = fileUtil.getFileFormatStr(context),
-            isMcoolFile = fileFormat === 'mcool',
-            isBWFile    = (fileFormat === 'bw' || fileFormat === 'bg'),
-            isBEDDBFile = (fileFormat === 'beddb' || fileFormat === 'bed'),
-            isBIGBEDFile = fileFormat === 'bigbed';
-        return isMcoolFile || isBWFile || isBEDDBFile || isBIGBEDFile;
+    /**
+    * Looks for static Higlass content in the context.
+    * @param {object} context
+    * @return {boolean} true if one static_content section has HiGlass viewconfs.
+    */
+    static hasHiglassItemInStaticContent(context){
```

Comment:
This function doesn't seem to be used?

Reply:
Yeah, I just noticed that when I started working on the next task. I'll delete it.